tests/utils/set_params.cpp

```diff
@@ -977,6 +1004,9 @@ void setStaggeredInvertParam(QudaInvertParam &inv_param)
   inv_param.input_location = QUDA_CPU_FIELD_LOCATION;
   inv_param.output_location = QUDA_CPU_FIELD_LOCATION;
 
+  // Gauge smear param
+  // setGaugeSmearParam(*inv_param.smear_param);
```

Comment:
Delete unnecessary comment

Reply:
removed in 1b58a101d2e0765827a3e26792cac35d57313ac3